pulp_container/app/viewsets.py

```diff
@@ -249,14 +251,34 @@ class TagViewSet(ContainerContentQuerySetMixin, ReadOnlyContentViewSet):
     }
 
 
-class ManifestViewSet(ContainerContentQuerySetMixin, ReadOnlyContentViewSet):
+class ManifestViewSetDetailMixin:
+    """
+    A mixin that allows differentiating between listing manifests and retrieving a single manifest.
```

Comment:
in the nutshell this is nice idea, in practice i think users will rely to see same fields for manifests whether on list of get. I.e. they will need to write 2 different parsers(a bit of inconvenience)  that will be able to cope with no labels/annotations fields during listing and take them into account when getting a specific instance.

Reply:
You do recommend returning `annotations` and `labels` for the list view too, right? Or, have you considered not exposing such fields at all?

Reply:
no no, i mean exposing them for list and get.
@mdellweg any opinion on the approach?

Reply:
This is a hard question that should ideally be answered on the application level. Do we promise our users in general that the objects from list and view calls are equivalent.

I fear that ship has sailed already. We do have similar things for tasks and rpm packages.

Reply:
I decided to use one and only `ManifestSerializer` to not do any other gimmicks. Thanks for your inputs!